### PR TITLE
Modified main page of Contributing section of ArviZ python library website

### PR DESCRIPTION
**Pull request by gayathri-krishnaswamy**
## Description
I had an offline discussion with @OriolAbril and made the following changes to the Contributing section of ArviZ python library website. I have introduced the following new sections to improve the clarity of the Contributing guide.

**Welcome** 
Provides a brief introduction to the project.
**Before you begin**
Provides prerequisites for new contributors.
**Communication**
Gitter channel details to help contributors get in touch with the community.
**Contributing to ArviZ**
Identifies multiple ways of contributing to ArviZ

### CODE REVIEW

File: doc/source/contributing/index.md [hunk 1]
```diff
@@ -1,50 +1,60 @@
...
-:caption: Contribution types overview
+:caption: Contribution types
 
 issue_reports
 translate
 issue_triaging
 outreach
 review_prs
 contributing_prs
+pr_tutorial
```
Comment (OriolAbril) [inline doc/source/contributing/index.md:57]:
```suggestion
```

This is already in the toctree below, I think pages should only appear once on the sidebar

File: doc/source/contributing/index.md [hunk 2]
```diff
@@ -1,50 +1,60 @@
...
-Further information on how to contribute to ArviZ can be found on these specific pages:
+### Review PRs
+[Review pull requests](https://arviz-devs.github.io/arviz/contributing/review_prs.html#review-prs) to ensure that the contributions are well tested and documented. For example, you can check if the documentation renders correctly and has no typos or mistakes. 
+
+### Manage issues
+Helping to manage or triage the open issues can be a great contribution and a great opportunity to learn about the various areas of the project. You can [triage existing issues](https://arviz-devs.github.io/arviz/contributing/issue_triaging.html#issue-triaging) to make them clear and/or provide temporary workarounds for them. 
+
+### Write documentation
+You can contribute to ArviZ’s documentation by either creating new content or editing existing content. For instance, you can add new ArviZ examples. To get involved with the documentation:
+1. Familiarize yourself with the [documentation content structure](https://arviz-devs.github.io/arviz/contributing/content_structure.html) and the [tool chain](https://arviz-devs.github.io/arviz/contributing/doc_toolchain.html) involved in creating the website.
```
Comment (OriolAbril) [inline doc/source/contributing/index.md:32]:
```suggestion
1. Familiarize yourself with the `{ref}`documentation content structure <content_structure>` and the {ref}`tool chain <doc_toolchain>` involved in creating the website.
```

Also necessary to add `(doc_toolchain)=` at the top of the `doc_toolchain.md` file.

File: doc/source/contributing/index.md [hunk 3]
```diff
@@ -1,50 +1,60 @@
...
+2. Understand the basic workflow for opening a [pull request](https://arviz-devs.github.io/arviz/contributing/pr_tutorial.html) and [reviewing changes](https://arviz-devs.github.io/arviz/contributing/review_prs.html).
```
Comment (OriolAbril) [inline doc/source/contributing/index.md:33]:
```suggestion
2. Understand the basic workflow for opening a {ref}`pull request <pr_tutorial>` and {ref}`reviewing changes <review_prs>`.
```

`(pr_tutorial)=` needs to be added to `pr_tutorial.md` file.

File: doc/source/contributing/index.md [hunk 4]
```diff
@@ -1,50 +1,60 @@
...
+3. Work on your content.
+4. [Build](https://arviz-devs.github.io/arviz/contributing/sphinx_doc_build.html) your documentation and preview the doc changes.
```
Comment (OriolAbril) [inline doc/source/contributing/index.md:35]:
```suggestion
4. {ref}`Build <sphinx_doc_build>` your documentation and preview the doc changes. 
```

File: doc/source/contributing/index.md [hunk 5]
```diff
@@ -1,50 +1,60 @@
...
+
+### Write documentation
+You can contribute to ArviZ’s documentation by either creating new content or editing existing content. For instance, you can add new ArviZ examples. To get involved with the documentation:
+1. Familiarize yourself with the [documentation content structure](https://arviz-devs.github.io/arviz/contributing/content_structure.html) and the [tool chain](https://arviz-devs.github.io/arviz/contributing/doc_toolchain.html) involved in creating the website.
+2. Understand the basic workflow for opening a [pull request](https://arviz-devs.github.io/arviz/contributing/pr_tutorial.html) and [reviewing changes](https://arviz-devs.github.io/arviz/contributing/review_prs.html).
+3. Work on your content.
+4. [Build](https://arviz-devs.github.io/arviz/contributing/sphinx_doc_build.html) your documentation and preview the doc changes. 
+   
+### Support outreach initiatives
+Support ArviZ [outreach initiatives](https://arviz-devs.github.io/arviz/contributing/outreach.html) such as writing blog posts, case studies,  getting people to use ArviZ at your company or university, etc.
```
Comment (OriolAbril) [inline doc/source/contributing/index.md:38]:
```suggestion
Support ArviZ {ref}`outreach initiatives <oureach_contrib>` such as writing blog posts, case studies,  getting people to use ArviZ at your company or university, etc.
```

File: doc/source/contributing/index.md [hunk 6]
```diff
@@ -1,50 +1,60 @@
...
+
+### Make changes in the code
+Fix outstanding issues (bugs) in the existing codebase. The issue can range from low-level software bugs to high-level design problems. You can also add new features to the codebase or improve the existing functionality. To fix bugs or add new features, do the following:
+1. Familiarize yourself with the [guidelines](https://arviz-devs.github.io/arviz/contributing/contributing_prs.html#steps-before-starting-work) before starting your work.
```
Comment (OriolAbril) [inline doc/source/contributing/index.md:42]:
```suggestion
1. Familiarize yourself with the {ref}`guidelines <steps_before_working>` before starting your work.
```

needs to have `(steps_before_working)=` added right above the `
## Steps before starting work` line in `contributing_prs.md`

File: doc/source/contributing/index.md [hunk 7]
```diff
@@ -1,50 +1,60 @@
...
+2. Understand the [development process](https://arviz-devs.github.io/arviz/contributing/contributing_prs.html#development-process-summary) and code conventions.
```
Comment (OriolAbril) [inline doc/source/contributing/index.md:43]:
```suggestion
2. Understand the {ref}`development process <dev_summary>` and code conventions.
```

File: doc/source/contributing/index.md [hunk 8]
```diff
@@ -1,50 +1,60 @@
...
+3. Understand the basic workflow for opening a [pull request](https://arviz-devs.github.io/arviz/contributing/pr_tutorial.html) and [reviewing](https://arviz-devs.github.io/arviz/contributing/review_prs.html) changes.
```
Comment (OriolAbril) [inline doc/source/contributing/index.md:44]: same references as before

File: doc/source/contributing/index.md [hunk 9]
```diff
@@ -1,50 +1,60 @@
...
-[ArviZ governance](https://github.com/arviz-devs/arviz/blob/main/GOVERNANCE.md)
-without having contributed code to ArviZ.
-All contributions to ArviZ, either on its codebase, its documentation,
-or its community are valuable and we really appreciate your help.
-
-:::{tip}
-Contact us on [Gitter](https://gitter.im/arviz-devs/community) if you want to
-contribute to the project but you are not sure where you can contribute or how to start.
-:::
+## Welcome
```
[the remark this answers is not in the document]
Comment (gayathri-krishnaswamy) [reply doc/source/contributing/index.md:2]: Fixed

File: doc/source/contributing/index.md [hunk 10]
```diff
@@ -1,50 +1,60 @@
...
+Welcome to the [ArviZ](https://www.arviz.org/en/latest/) project! If you’re reading this guide, that probably means you’d like to get involved and start contributing to the project. ArviZ is a  multi-language open-source project that provides tools for exploratory analysis of Bayesian models.   The package includes functions for posterior analysis, data storage, sample diagnostics, model checking, and comparison.
```
[the remark this answers is not in the document]
Comment (gayathri-krishnaswamy) [reply doc/source/contributing/index.md:3]: Fixed

File: doc/source/contributing/index.md [hunk 11]
```diff
@@ -1,50 +1,60 @@
...
+
+As a scientific, community-driven, open-source software project, we welcome contributions from interested individuals or groups. The guidelines specified in this document can help new contributors make their contributions compliant with the conventions of the ArviZ python library, and maximize the probability of such contributions being merged as quickly and as efficiently as possible.
+
+All kinds of contributions to ArviZ, either on its codebase, its documentation, or its community are valuable and we appreciate your help. We constantly encourage non-code contributors to become a core contributor and participate in [ArviZ governance](https://github.com/arviz-devs/arviz/blob/main/GOVERNANCE.md).
```
[the remark this answers is not in the document]
Comment (gayathri-krishnaswamy) [reply doc/source/contributing/index.md:7]: Fixed

File: doc/source/contributing/index.md [hunk 12]
```diff
@@ -1,50 +1,60 @@
...
+ 
+## Communication
+Contact us on [Gitter](https://gitter.im/arviz-devs/community) if you want to contribute to the project but you are not sure where you can contribute or how to start.
+
+## Contributing to ArviZ
+The most common way of contributing to the project is sending pull requests on the main [ArviZ Github repository](https://github.com/arviz-devs/arviz). You can contribute to ArviZ in the following ways:
 
-Below we list some examples of contributions that might serve as inspiration.
+### Create an issue
+[Submit issues](https://github.com/arviz-devs/arviz/issues/new/choose) for existing bugs or desired enhancements. Check [Issue reports](https://arviz-devs.github.io/arviz/contributing/issue_reports.html) for details on how to write an issue.
```
[the remark this answers is not in the document]
Comment (gayathri-krishnaswamy) [reply doc/source/contributing/index.md:19]: Fixed

File: doc/source/contributing/index.md [hunk 13]
```diff
@@ -1,50 +1,60 @@
...
-* Translate our website to languages other than English. Check the details at {ref}`translate`
-* Fixing outstanding [issues](https://github.com/arviz-devs/arviz/issues) (bugs) with the existing codebase. They range from low-level software bugs to higher-level design problems.
-* Contributing or improving the {ref}`documentation <homapage>` (`docs`) or examples (`arviz/examples`).
-* Help users submitting issues by making sure their issue is clear and/or providing temporal
-  workarounds for them. Check the details at {ref}`issue_triaging`
-* Review Pull Requests ensuring contributions are well tested and documented, that documentation
-  renders correctly and has no typos or mistakes. Check the details at {ref}`review_prs`
-* Adding new or improved functionality to the existing codebase.
+### Translate ArviZ website
+You can [translate](https://python.arviz.org/en/latest/contributing/translate.html#translate) our website to languages other than english.
```
[the remark this answers is not in the document]
Comment (gayathri-krishnaswamy) [reply doc/source/contributing/index.md:22]: Fixed

File: doc/source/contributing/index.md (hunk 8 above)
Comment (gayathri-krishnaswamy) [reply doc/source/contributing/index.md:44]: Fixed

File: doc/source/contributing/index.md [hunk 14]
```diff
@@ -1,50 +1,60 @@
...
-* Adding new or improved functionality to the existing codebase.
+### Translate ArviZ website
+You can [translate](https://python.arviz.org/en/latest/contributing/translate.html#translate) our website to languages other than english. 
 
-Further information on how to contribute to ArviZ can be found on these specific pages:
+### Review PRs
+[Review pull requests](https://arviz-devs.github.io/arviz/contributing/review_prs.html#review-prs) to ensure that the contributions are well tested and documented. For example, you can check if the documentation renders correctly and has no typos or mistakes. 
+
+### Manage issues
+Helping to manage or triage the open issues can be a great contribution and a great opportunity to learn about the various areas of the project. You can [triage existing issues](https://arviz-devs.github.io/arviz/contributing/issue_triaging.html#issue-triaging) to make them clear and/or provide temporary workarounds for them.
```
[the remark this answers is not in the document]
Comment (gayathri-krishnaswamy) [reply doc/source/contributing/index.md:28]: Fixed

File: doc/source/contributing/index.md (hunk 1 above)
Comment (gayathri-krishnaswamy) [reply doc/source/contributing/index.md:57]: Fixed

File: doc/source/contributing/index.md (hunk 2 above)
Comment (gayathri-krishnaswamy) [reply doc/source/contributing/index.md:32]: Fixed

File: doc/source/contributing/index.md (hunk 3 above)
Comment (gayathri-krishnaswamy) [reply doc/source/contributing/index.md:33]: Fixed

**Comment by OriolAbril** [comment]
Thanks @gayathri-krishnaswamy !

**Pull request merged**